packages/form-js-editor/src/features/properties-panel/PropertiesPanelHeaderProvider.js

```diff
@@ -54,10 +40,17 @@ export const PropertiesPanelHeaderProvider = {
       type
     } = field;
 
-    const Icon = iconsByType(type);
+    // @Note: We know that we are inside the properties panel context,
+    // so we can savely use the hook here.
+    // eslint-disable-next-line
```

Comment:
Maybe we should disable a specific rule in the next line instead of disabling ESLint completely, no?